src/pykiso/lib/connectors/cc_proxy.py

```diff
@@ -60,28 +60,28 @@ def _cc_close(self) -> None:
         self.queue_out = Queue()
         log.debug("Close proxy channel")
 
-    def _cc_send(self, *args: tuple, **kwargs: dict) -> None:
+    def _cc_send(self, any_msg: Any, raw: bool = False) -> None:
         """Populate the queue in of the proxy connector.
 
         :param args: tuple containing positionnal arguments
         :param kwargs: dictionary containing named arguments
         """
-        log.debug(f"put at proxy level: {args} {kwargs}")
-        self.queue_in.put((args, kwargs))
+        log.debug(f"put at proxy level: {any_msg} {raw}")
+        self.queue_in.put((any_msg, raw))
```

Comment:
I think we should pass args, kwargs in every queue. Then we would be fully data agnostic.
raw or remote_id can then be part of the args or kwars.